test/alternator/test_metrics.py

```diff
@@ -138,6 +146,25 @@ def test_batch_get_item_count(test_table_s, metrics):
         test_table_s.meta.client.batch_get_item(RequestItems = {
             test_table_s.name: {'Keys': [{'p': random_string()}, {'p': random_string()}], 'ConsistentRead': True}})
 
+KB = 1024
+def test_rcu(test_table_s, metrics):
+    with check_increases_table_metric(metrics, 'scylla_column_family_cru_total', test_table_s.name, 4):
+        p = random_string()
+        val = random_string()
+        total_length = len(p) + len(val) + len("pattanother")
+        val2 = 'a' * (4 * KB - total_length + 1) # message length 4KB +1
+
+        test_table_s.put_item(Item={'p': p, 'att': val, 'another': val2})
+        test_table_s.get_item(Key={'p': p}, ConsistentRead=True)
+
+def test_wcu(test_table_s, metrics):
+    with check_increases_table_metric(metrics, 'scylla_column_family_cwu_total', test_table_s.name, 6):
+        p = random_string()
+        val = random_string()
+        total_length = len(p) + len(val) + len("pattanother")
+        val2 = 'a' * (2 * KB - total_length + 1) # message length 2K + 1
+        test_table_s.put_item(Item={'p': p, 'att': val, 'another': val2})
```

Comment:
A thought (not necessary and certainly not now): Would be nice to check that a put_item, even if it needs to read the old value of the item, does NOT increment the rcu counter.

Reply:
it's a good idea, as I'll add more functionality I'll add more tests